app/views/idv/doc_auth/verify.html.erb

```diff
@@ -39,14 +39,25 @@
     ) %>
   </div>
   <%= "#{t('doc_auth.forms.ssn')}: #{flow_session[:pii_from_doc][:ssn]}" %>
-  <div class='mt4'>
+  <%= render 'shared/spinner-button',
+             action_message: t('doc_auth.info.verifying'),
+             class: 'mt4' do %>
     <%= button_to(
       t('forms.buttons.continue'),
       url_for,
       method: :put,
+      form: {
+        class: 'button_to read-after-submit',
```

Comment:
Note to self: Research why [accessible-forms.js](https://github.com/18F/identity-idp/blob/master/app/javascript/app/accessible-forms.js) exists, since in cases like this it prevents screen readers from announcing intended status texts after a submission. At the very least, it could be improved to respect `event.defaultPrevented` in a `submit` event.